extras/ai-battle/HeadlessGame.h

```diff
@@ -0,0 +1,49 @@
+// Copyright (C) 2005 - 2024 Settlers Freaks (sf-team at siedler25.org)
+//
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "Game.h"
+#include "Replay.h"
+#include "ai/AIPlayer.h"
+#include "gameTypes/AIInfo.h"
+
+#include <boost/filesystem.hpp>
+
```

Comment:
Newlines not required
```suggestion
#include <boost/filesystem.hpp>
```